tests/utils/k8s.go

```diff
@@ -43,7 +46,22 @@ func DropNamespace(clientSet *kubernetes.Clientset, name string) error {
 		PropagationPolicy:  &propagationPolicy,
 	}
 
-	return clientSet.CoreV1().Namespaces().Delete(name, &options)
+	err := clientSet.CoreV1().Namespaces().Delete(name, &options)
+	if err != nil {
+		return err
+	}
+
+	return retry(namespaceDeletionTimeout, 3*time.Second, func() error {
+		_, err := clientSet.CoreV1().Namespaces().Get(name, metav1.GetOptions{})
+		if err == nil {
+			return errors.New("the namespace is still there")
+		} else if statusErr, ok := err.(*apiErrors.StatusError); !ok || statusErr.Status().Reason != metav1.StatusReasonNotFound {
+			return err
+		} else {
+			log.Info("Deleted!")
```

Comment:
can we make log more informative e.g. `Namespace <namespace name> deleted!`?

Reply:
done